tests/src/test/scala/cats/tests/TrySuite.scala

```diff
@@ -17,7 +17,11 @@ class TrySuite extends CatsSuite {
   checkAll("Try[Int]", CoflatMapTests[Try].coflatMap[Int, Int, Int])
   checkAll("CoflatMap[Try]", SerializableTests.serializable(CoflatMap[Try]))
 
-  checkAll("Try with Throwable", MonadErrorTests[Try, Throwable].monadError[Int, Int, Int])
+  //temporarily disable this test due to scala.util.Failure regression https://github.com/scala/bug/issues/11242
+  if (BuildInfo.scalaVersion != "2.13.0-M5") {
```

Comment:
We might want to open an issue in this repo to remind ourselves to remove this hack.

Reply:
done: #2601

Reply:
thanks @erikerlandson